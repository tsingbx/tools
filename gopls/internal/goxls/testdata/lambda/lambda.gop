var myString string //@myString

func onStart(fn func(int) int) { //@onStart
	println fn(100), myString //@godef("myString", myString)
}

onStart xparam => { //@xparam,godef("onStart",onStart)
	return xparam * xparam //@godef("xparam",xparam)
}

onStart yparam => yparam //@yparam,godef("yparam",yparam)
