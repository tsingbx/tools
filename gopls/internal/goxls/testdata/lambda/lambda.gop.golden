-- myString-hoverdef --
```go
var myString string
```

@myString

-- onStart-hoverdef --
```go
func onStart(fn func(int) int)
```
-- xparam-hoverdef --
```go
var xparam int
```
-- yparam-hoverdef --
```go
var yparam int
```
